### PR TITLE
New contents.

### PR DESCRIPTION
**Pull request by kaikai1024**
* Rename hacking to advanced-use.
* Remove the compile-cita.
* Add ops.

preview:

2019-09-18-170635_273x370_scrot

TODO:

* Translate the content
* Add the `exception-handle` and `service-monitor`

### CODE REVIEW

File: website/sidebars.json [hunk 1]
```diff
@@ -88,16 +88,23 @@
       },
       "cita/store",
       {
-        "label": "Hacking",
+        "label": "Advanced-use",
```
Comment (leeyr338) [inline website/sidebars.json]: Is it better to use Chinese?

File: website/sidebars.json [hunk 2]
```diff
@@ -88,16 +88,23 @@
...
-          "cita/hacking/contracts/batch-tx"
+          "cita/advanced-use/contracts/solidity",
+          "cita/advanced-use/contracts/rust",
+          "cita/advanced-use/auto-exec",
+          "cita/advanced-use/chain-gov",
+          "cita/advanced-use/contracts/batch-tx"
+        ]
+      },
+      {
+        "label": "Ops",
```
Comment (leeyr338) [inline website/sidebars.json:102]: Is it better to use Chinese?

Comment (kaikai1024) [reply website/sidebars.json:102]: Check the `TODO` part. 

https://github.com/cryptape/citahub-docs/blob/master/website/sidebars.json

File: website/sidebars.json (hunk 1 above)
Comment (kaikai1024) [reply website/sidebars.json]: https://github.com/cryptape/citahub-docs/blob/master/website/sidebars.json

**Review by leeyr338 — APPROVED** [review]

File: website/sidebars.json (hunk 2 above)
Comment (leeyr338) [reply website/sidebars.json:102]: Got it.

Comment (kaikai1024) [reply website/sidebars.json:102]: The chinese version has to be uploaded using crowdin.

**Review by nanmuxiusi — APPROVED** [review]

**Pull request merged**